server/services/offendersService.js

```diff
@@ -32,50 +32,34 @@ function isOverdue(dbDate) {
   return date.isBefore(moment(0, 'HH'))
 }
 
-async function getSentenceMap(offenderList, nomisClient) {
+async function getSentenceMap(offenderList, prisonerSearchClient) {
   const bookingIds = offenderList
     .filter(o => !o.dbRecord || !o.dbRecord.catType || o.dbRecord.catType === CatType.INITIAL.name)
     .map(o => o.bookingId)
 
-  const sentenceDates = await nomisClient.getSentenceDatesForOffenders(bookingIds)
+  const prisoners = await prisonerSearchClient.getPrisonersByBookingIds(bookingIds)
 
   return new Map(
-    sentenceDates
-      .filter(s => s.sentenceDetail.sentenceStartDate) // the endpoint returns records for offenders without sentences
-      .map(s => {
-        const { sentenceDetail } = s
-        return [sentenceDetail.bookingId, { sentenceDate: sentenceDetail.sentenceStartDate }]
-      })
+    prisoners
+      .filter(s => s.sentenceStartDate) // the endpoint returns records for offenders without sentences
+      .map(s => [s.bookingId, { sentenceDate: s.sentenceStartDate }])
```

Comment:
I could have gone further and referred to sentenceStartDate in calls raher than sentenceDate